blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -116,4 +118,4 @@ PipelinePage.childContextTypes = {
 const selectors = createSelector([pipelineSelector],
     (pipeline) => ({ pipeline }));
 
-export default connect(selectors, actions)(documentTitle(PipelinePage));
+export default translate(['jenkins.plugins.blueocean.dashboard.Messages'], { wait: true })(connect(selectors, actions)(documentTitle(PipelinePage)));
```

Comment:
@michaelneale I think that's a react "pattern" for kinda mixing functionality i.e. I think  it creates a composite component in the BG... funky stuff.

Maybe we can eliminate this with an annotation type thing e.g. `@Translation('jenkins.plugins.blueocean.dashboard.Messages')` + some build time tooling?